servers/partner-ci/7.0/contrail.yaml

```diff
@@ -2,70 +2,40 @@
 - parameter:
     name: plugin-specific-parameters
     parameters:
-      - bool:
-          name: UPDATE_MASTER
-          default: '{update_master}'
-          description: 'turns on update fuel master node to maintenance update'
-      - bool:
-          name: BONDING
-          default: '{obj:bonding}'
-      - bool:
-          name: KVM_USE
-          default: '{obj:kvm_use}'
-      - string:
-          name: NODE_VOLUME_SIZE
-          default: '512'
-      - string:
-          name: MIRROR
-          default: 'http://mirror.seed-cz1.fuel-infra.org'
-          description: 'mirror for package repositories. this mirror is optimized for CZ'
-      - string:
-          name: MIRROR_UBUNTU
-          default: >
-             deb ${{MIRROR}}/pkgs/ubuntu/ trusty main universe multiverse |
-             deb ${{MIRROR}}/pkgs/ubuntu/ trusty-updates main universe multiverse
-          description: 'proposed repositories to update ubuntu cluster'
-      - string:
-          name: CONTRAIL_DISTRIBUTION
-          default: ''
-      - string:
-          name: CONTRAIL_VERSION
-          default: '{plugin_version}'
-          description: >
-            'There is a version of juniper packages stored in common sotrage'
-      - string:
-          name: POOL_ADMIN
-          default: '10.109.0.0/24:24'
-      - string:
-          name: POOL_PUBLIC
-          default: '10.109.1.0/24:24'
-      - string:
-          name: POOL_MANAGEMENT
-          default: '10.109.2.0/24:24'
-      - string:
-          name: POOL_PRIVATE
-          default: '10.109.3.0/24:24'
-      - string:
-          name: POOL_STORAGE
-          default: '10.109.4.0/24:24'
-      - string:
-          name: PLUGIN_VERSION
-          default: ${{CONTRAIL_VERSION}}
-          description: 'plugin version from metadata'
-      - string:
-          name: CONTRAIL_PLUGIN_PATH
-          description: 'The path to the contrail package on storage'
-      - string:
-          name: CONTRAIL_PLUGIN_PACK_UB_PATH
-          description: 'The path to the contrail package on storage'
-      - string:
-          name: VSRX_TEMPLATE_PATH
-          default: '/storage/contrail/vSRX1.template'
-          description: 'The path to the vsrx template'
-
+      - text:
+          name: SPECIFIC_PROPS
+          default: |
+            POOL_STORAGE=10.109.4.0/24:24;
+            POOL_PRIVATE=10.109.3.0/24:24;
+            POOL_MANAGEMENT=10.109.2.0/24:24;
+            POOL_ADMIN=10.109.0.0/24:24;
+            BM_HOST_BRIDGE_INTERFACES=eth2:10.109.0.0/24;eth3:10.109.4.0/24;
```

Comment:
1. move baremetal to the baremetal template as properties
2. we should solve a problem with **;** in variables, because it affects parser (also the problem could be solved by changing `IFS=";"` to`IFS=";\n"`, but need to test that